client/my-sites/upgrades/checkout/privacy-protection.jsx

```diff
@@ -73,6 +76,33 @@ module.exports = React.createClass( {
 			priceForPrivacyText = this.translate( 'Privacy protection keeps your information hidden when these domains are searched for in the public database. You can use it for no extra cost.' );
 		}
 
+		if ( abtest( 'privacyCheckbox' ) === 'checkbox' ) {
+			return (
+				<div>
+					<Card className="checkout__privacy-protection-checkbox">
```

Comment:
Ideally, this would be its own component, but depending on the results of this test, this may replace the rest of `PrivacyProtection`, so I think it makes sense to leave it like this for the test.